pubspec.lock

```diff
@@ -210,6 +100,20 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.8.0-nullsafety.2"
+  sqflite:
```

Comment:
Is this supposed to be changed to shared preference in issue #8 ?

Reply:
Yes, this is old code.